README.md

```diff
@@ -155,6 +155,10 @@ mopp align -i ./test/data/out3/cat \
 
 `mopp align` aligns the sequencing data provided in the input directory to the reference index. Providing a file pattern `-p` allows for specification of files with certain name patterns. Allocating more threads to this command `-t` can reduce processing time.
 
+The index `-x` is created by the bowtie2-build command. This command outputs the forward (.bt2) and reverse (rev.bt2) index files. Our parameter requests the common prefix that is shared by all these files, before the forward/reverse designation.
```

Comment:
This is good. Could you specify `This command` refers to the bowtie2-build instead of the mopp align command to avoid confusion?

Could you clarify the structure of `-x` command, which is the path to the index folder plus index prefix?